SPEC.md

```diff
@@ -85,20 +85,40 @@ The operations that CNI plugins must support are:
   - All parameters should be the same as those passed to the corresponding add operation.
   - A delete operation should release all resources held by the supplied containerid in the configured network.
 
-- Report version
+- `GET`: Get container network configuration
+  - This action should return the same result as an `ADD` action for the same inputs. The result should not change over the lifetime of the container.
+  - The plugin should return an error if any general internal state is unexpected. For example, if the plugin's data storage is missing or corrupt, or it cannot talk to a controller (if it has one) controller, it should return an error.
+  - The plugin should not return an error if its expected sandbox state (eg interfaces, IP addresses, routes, etc) is not found, as subsequent plugins in the chain may alter this state.
+  - The plugin will be passed a `prevResult` field in its configuration JSON on stdin which contains the `Result` object which the plugin (or the chain of which the plugin is part) returned to the runtime 
+  regenerate as much of its `Result` object as it can, which may be modified by plugins in the chain. For example, if the plugin-created interface exists and has the expected IP address, the plugin should add the interface and address to its `Result`. If the plugin-created interface does not have the expected IP address, the plugin should add the interface (but not the expected IP address) to its `Result`.
+  - A runtime may call `GET` at any time; but if `GET` is called for a given container before an `ADD` or after a `DEL` for that container, the plugin should return error 3 to indicate the container is unknown (see [Well-known Error Codes](#well-known-error-codes) section).
+  - 
+  - Parameters:
+    - **Container ID**, as defined for `ADD`.
+    - **Network namespace path**, as defined for `ADD`.
+    - **Network configuration**, as defined for `ADD`.
```

Comment:
Is it worth mentioning that prevResult is also set for the first plugin in the chain?

Reply:
@matthewdupre yes, it would be.